tests/issues/issue-618.php

```diff
@@ -652,14 +652,14 @@
     'issue-618-M' => [
         'ua' => 'MobileSafari/600.1.4 CFNetwork/711.1.16 Darwin/14.0.0',
```

Comment:
Here's an example where we shouldn't see Safari 4.0 on iOS 8.1.  The previous patterns were looking at the safari build number to decide if it was Safari 4.0, or 6.0...  but in the UAs I was seeing, that build number doesn't vary that much (for mobile safari), and it was detecting a lot of UAs as 4.0:

```
MobileSafari/600.1.4 CFNetwork/711.0.6 Darwin/14.0.0
MobileSafari/600.1.4 CFNetwork/711.1.12 Darwin/14.0.0
MobileSafari/600.1.4 CFNetwork/711.1.16 Darwin/14.0.0
MobileSafari/600.1.4 CFNetwork/711.3.18 Darwin/14.0.0
MobileSafari/601.1 CFNetwork/758.2.8 Darwin/15.0.0
MobileSafari/601.1 CFNetwork/758.4.3 Darwin/15.5.0
MobileSafari/601.1 CFNetwork/758.5.3 Darwin/15.6.0
MobileSafari/602.1 CFNetwork/808.0.2 Darwin/16.0.0
MobileSafari/602.1 CFNetwork/808.1.4 Darwin/16.1.0
MobileSafari/602.1 CFNetwork/808.2.16 Darwin/16.3.0
MobileSafari/602.1 CFNetwork/808.3 Darwin/16.3.0
MobileSafari/602.1 CFNetwork/811.4.18 Darwin/16.5.0
MobileSafari/602.1 CFNetwork/811.5.1 Darwin/16.6.0
MobileSafari/602.1 CFNetwork/811.5.4 Darwin/16.6.0
MobileSafari/602.1 CFNetwork/811.5.4 Darwin/16.7.0
MobileSafari/604.1 CFNetwork/887 Darwin/17.0.0
```